functions/Restore-DbaDatabase.ps1

```diff
@@ -426,7 +426,7 @@ function Restore-DbaDatabase {
                                     $backupFiles += $p
                                 }
                                 else {
-                                    Write-Warning "$FunctionName - $p cannot be accessed by $SqlInstance"
+                                    Write-Message -Level Warning -Message "$FunctionName - $p cannot be accessed by $SqlInstance" 
                                     continue
```

Comment:
Stop-Function is designed to handle this.
 - Use `-Continue` parameter to process next item
 - Use `-SilentlyContinue` parameter if you want it to continue even in unattended mode (e.g. with scripts calling it. They won't have a way to know this failed if you do.